sys/bloom/bloom.c

```diff
@@ -35,7 +35,7 @@ bloom_t *bloom_new(size_t size, size_t num_hashes, ...)
     }
 
     /* Allocate Bloom array */
-    if (!(bloom->a = calloc(ROUND(size), sizeof(char)))) {
+    if (!(bloom->a = calloc(1, ROUND(size)))) {
```

Comment:
The prototype is `void *calloc(size_t number_of_items, size_t size_of_an_item)`, and scan-build issued a warning that the element size was wrong